package.json

```diff
@@ -4,7 +4,7 @@
   "private": true,
   "scripts": {
     "start": "node app.js",
-    "test": "eslint app.js controllers lib && LEVERAGE_API_CFG=test/config.json mocha -R spec",
+    "test": "eslint app.js controllers lib && npm run test-data && LEVERAGE_API_CFG=test/config.json mocha -R spec",
```

Comment:
This line here, was why I had the `config.test.json` file from PR #44 :) I was playing with different configurations between test and live locally.